src/vw/src/test/scala/VerifyVowpalWabbitClassifier.scala

```diff
@@ -0,0 +1,147 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import java.io.File
+
+import org.apache.spark.ml.evaluation.{BinaryClassificationEvaluator}
+import org.apache.spark.ml.util.MLReadable
+import org.apache.spark.sql.DataFrame
+import java.nio.file.{Files, Path, Paths}
+
+import org.apache.spark.sql.functions._
+import org.apache.commons.io.FileUtils
+import org.apache.spark.ml.feature.{Binarizer, StringIndexer}
+import org.apache.spark.ml.tuning.{CrossValidator, ParamGridBuilder, TrainValidationSplit}
+
+class VerifyVowpalWabbitClassifier extends Benchmarks with EstimatorFuzzing[VowpalWabbitClassifier] {
+  lazy val moduleName = "vw"
+  val numPartitions = 2
+
+  test("Verify VowpalWabbit Classifier can be run with TrainValidationSplit") {
+      val fileName = "a1a.train.svmlight"
+
+      val fileLocation = DatasetUtils.binaryTrainFile(fileName).toString
+      val dataset = session.read.format("libsvm").load(fileLocation).repartition(numPartitions)
+
+    val vw = new VowpalWabbitClassifier()
+        .setArgs("--link=logistic --quiet")
+
+    val paramGrid = new ParamGridBuilder()
+       .addGrid(vw.learningRate, Array(0.5, 0.05, 0.001))
+       .addGrid(vw.numPasses, Array(1, 3, 5))
+       .build()
+
+    val cv = new CrossValidator()
+      .setEstimator(vw)
+      .setEvaluator(new BinaryClassificationEvaluator)
+      .setEstimatorParamMaps(paramGrid)
+      .setNumFolds(3)
+      .setParallelism(2) // thanks to dynamic port assignment in spanning_tree.cc :)
+
+    val model = cv.fit(dataset)
+    model.transform(dataset)
+
+    val auc = model.avgMetrics(0)
+    println(s"areaUnderROC: $auc")
+
+    // TODO: what the best approach here? pass all individual parameters or the training args or both?
```

Comment:
sorry not sure what this is asking, could you please clarify?  pass the training args where?

Reply:
what is difference between individual params and training args?

Reply:
I think this was related to some intermediate code. not relevant anymore, thus removed the code.